source/braille.py

```diff
@@ -1974,6 +1994,49 @@ def _get_scriptableObject(self):
 			return display
 		return super(BrailleDisplayGesture, self).scriptableObject
 
+	def _get_script(self):
+		# Overrides L{inputCore.InputGesture._get_script} to support modifier keys.
+		script=scriptHandler.findScript(self)
+		if script:
+			self.script = script
+			return self.script
+		# No script for this gesture has been found, so process this gesture for possible modifiers. 
+		# For example, if L{self.id} is 'key1+key2',
+		# key1 is bound to 'kb:control' and key2 to 'kb:tab',
+		# this gesture should execute 'kb:control+tab'.
+		# Combining modifiers with braille input (#7306) is not yet supported.
+		gestureKeys = set(self.keyNames)
+		gestureModifiers = set()
+		for keys, modifiers in handler.display._getModifierGestures():
+			if keys<gestureKeys:

```

Comment:
@dkager: see this line and lines below, which contain some set operator stuff.

Reply:
I assume `<` means "subset of", but I'm not sure. That's why I am wondering if the benefit of overloaded operators is really a benefit.

Reply:
Exactly.